tests/ui/impl-trait/in-trait/method-signature-matches.stderr

```diff
@@ -59,6 +65,11 @@ LL |     fn come_on_a_little_more_effort(_: (), _: (), _: ()) -> impl Sized;
 ...
 LL |     fn come_on_a_little_more_effort() {}
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 3 parameters, found 0
+   |
+help: modify the signature to match the trait definition
+   |
+LL |     (), _: (), _: () {}
```

Comment:
What happened here?

Reply:
oh. That's a bug of my implementation which doesn't consider the impl function being 0 parameters. I'll fix it